cvat/rq_patching.py

```diff
@@ -77,7 +80,72 @@ def custom_started_job_registry_cleanup(self, timestamp: Optional[float] = None)
     return job_ids
 
 
-def update_started_job_registry_cleanup() -> None:
+def cancel_job(self: Job, pipeline: Pipeline | None = None, enqueue_dependents: bool = False):
```

Comment:
It makes sense to add a comment about what's changed, why, and when it can be removed from the codebase.

Reply:
I believe a link to the PR with description is enough